src/main/webapp/app/shared/sidebar/sidebar-card-small/sidebar-card-small.component.html

```diff
@@ -1,9 +1,20 @@
+<!-- Explicitly implemented query parameters for the messages module as a workaround, since routing in the messages module
+ operates differently over the MetisConversations service, it will get adjusted in a followup PR -->
 <div
     id="test-sidebar-card"
     class="pointer rounded-3 col-12 border-start border-5 highlight-card bg-module padding-small"
-    [routerLink]="itemSelected ? '../' + sidebarItem?.id : './' + sidebarItem?.id"
+    [ngClass]="{
+        'border-primary': isConversationUnread
```

Comment:
Please avoid function calls in the HTML template, this includes getter methods, too.